tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -522,6 +522,100 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
 					})
 				})
 
+				When("a Redis instance is created with a specific name and json output", func() {
+
+					var name string
+					var svcFullName string
+					var output string
+
+					BeforeEach(func() {
+						name = helper.RandString(6)
+						svcFullName = strings.Join([]string{"Redis", name}, "/")
+						output = helper.Cmd("odo", "service", "create", fmt.Sprintf("%s/Redis", redisOperator), name, "--project", commonVar.Project, "-o", "json").ShouldPass().Out()
+					})
+
+					AfterEach(func() {
+						helper.Cmd("odo", "service", "delete", svcFullName, "-f").ShouldRun()
+					})
+
+					It("should display valid information in output of create command", func() {
+						By("displaying service information", func() {
+							values := gjson.GetMany(output, "kind", "metadata.name", "manifest.kind", "manifest.metadata.name")
+							expected := []string{"Service", "Redis/" + name, "Redis", name}
+							Expect(helper.GjsonMatcher(values, expected)).To(Equal(true))
+						})
+
+						By("not containing cluster specific information", func() {
+							values := gjson.GetMany(output, "inDevfile", "deployed", "manifest.metadata.creationTimestamp")
+							expected := []string{"true", "false", ""}
+							Expect(helper.GjsonMatcher(values, expected)).To(Equal(true))
+						})
+					})
+
+					When("executing odo service describe", func() {
+						var descOutput string
+						BeforeEach(func() {
+							descOutput = helper.Cmd("odo", "service", "describe", "Redis/"+name, "--project", commonVar.Project, "-o", "json").ShouldPass().Out()
+						})
+
+						It("should display valid information", func() {
+							By("displaying service information", func() {
+								values := gjson.GetMany(descOutput, "kind", "metadata.name", "manifest.kind", "manifest.metadata.name")
+								expected := []string{"Service", "Redis/" + name, "Redis", name}
+								Expect(helper.GjsonMatcher(values, expected)).To(Equal(true))
+							})
+
+							By("not containing cluster specific information", func() {
+								values := gjson.GetMany(descOutput, "inDevfile", "deployed", "manifest.metadata.creationTimestamp")
+								expected := []string{"true", "false", ""}
+								Expect(helper.GjsonMatcher(values, expected)).To(Equal(true))
+							})
+						})
+					})
+
+					When("odo push is executed", func() {
+
+						BeforeEach(func() {
+							helper.Cmd("odo", "push").ShouldPass()
+						})
+
+						It("should describe cluster specific information in json format", func() {
+							descOutput := helper.Cmd("odo", "service", "describe", "Redis/"+name, "--project", commonVar.Project, "-o", "json").ShouldPass().Out()
+							values := gjson.GetMany(descOutput, "inDevfile", "deployed")
+							expected := []string{"true", "true"}
+							Expect(helper.GjsonMatcher(values, expected)).To(Equal(true))
+							tsValue := gjson.Get(descOutput, "manifest.metadata.creationTimestamp")
+							Expect(tsValue.Str).NotTo(BeEmpty())
+						})
```

Comment:
Same comment for this `It` block as above. Maybe add `By` statements to be inline with other tests? I also think that adding `By` statements makes this more readable.